### ISSUE TITLE
runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin#8915]
https://app.getsentry.com/red-hat/openshift-3-stg/issues/129289018/

```
runtime.errorString: runtime error: invalid memory address or nil pointer dereference
  File "/builddir/build/BUILD/atomic-openshift-git-0.a4463d9/_build/src/github.com/openshift/origin/pkg/cmd/util/serviceability/panic.go", line 27, in CapturePanic.fm
  File "/builddir/build/BUILD/atomic-openshift-git-0.a4463d9/_thirdpartyhacks/src/k8s.io/kubernetes/pkg/util/runtime/runtime.go", line 40, in HandleCrash
  File "/usr/lib/golang/src/runtime/asm_amd64.s", line 402, in call32
  File "/usr/lib/golang/src/runtime/panic.go", line 387, in gopanic
  File "/usr/lib/golang/src/runtime/panic.go", line 42, in panicmem
...
(17 additional frame(s) were not displayed)
```

**Comment by deads2k** [comment]
@smarterclayton did you push the button to allow sentry access to private repos and auto-create issues for you?

Also, I think I fixed this already with https://github.com/kubernetes/kubernetes/pull/25472.  You want it picked?  I was just going to wait.

**Comment by smarterclayton** [comment]
This is probably a backport - @brenton

**Comment by ncdc** [comment]
@deads2k reassigned to you

**Comment by deads2k** [comment]
@brenton which stream would you like me to backport this into?

**Comment by brenton** [comment]
I can't really judge the severity by stack trace.  Let's start with the ose/master branch since that's 3.2.1.

**Comment by smarterclayton** [comment]
It's in pod cleanup on a deleted node.  It might block releasing quota for pods in a user's namespace if a node goes down, thus resulting in the app not being able to scale up.

**Comment by brenton** [comment]
@smarterclayton, would anything in the system eventually recover from this situation or is this a bug that just needs to be fixed right away?

**Comment by smarterclayton** [comment]
It's possible that pods would be orphaned, but pod GC should eventually
clean it up.

On May 20, 2016, at 9:16 AM, Brenton Leanhardt <notifications@github.com>
wrote:

@smarterclayton <https://github.com/smarterclayton>, would anything in the
system eventually recover from this situation or is this a bug that just
needs to be fixed right away?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/issues/8915#issuecomment-220602505>

**Comment by brenton** [comment]
@abhgupta, are you OK with having this fix with 3.2.1?

**Comment by abhgupta** [comment]
@brenton 3.2.1 seems OK to me for this issue/fix.
